src/aiq/observability/register.py

```diff
@@ -15,8 +15,9 @@
 
 import logging
 import os
-
 from pydantic import Field
+from ragaai_catalyst import RagaAICatalyst
+from ragaai_catalyst.tracers.exporters import DynamicTraceExporter
```

Comment:
Can you move these lines inside of the `catalyst_telemetry_exporter` function? So we only import them when requested.